RecipeApp/RecipeApp/AllRecipeGridViewController.swift

```diff
@@ -33,16 +35,30 @@ class AllRecipeGridViewController: UIViewController, UICollectionViewDataSource,
         layout.itemSize = CGSize(width: width, height: width*1.2)
 
         //placeholder data structure holding recipes until we have api call to get this
-        for i in 1...20 {
-            //create recipe
-            let title = "Test \(i)"
-            let image = UIImage(named: "rice-png")!
-            let recipe = Recipe(givenTitle: title, givenImage: image)
-            recipes.append(recipe)
-            collectionView.reloadData()
-        }
+//        for i in 1...20 {
+//            //create recipe
+//            let title = "Test \(i)"
+//            let image = UIImage(named: "rice-png")!
+//            let recipe = Recipe(givenTitle: title, givenImage: image)
+//            recipes.append(recipe)
+//            collectionView.reloadData()
+//        }
     }
     
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        let query = PFQuery(className: "Recipe")
+        query.includeKeys(["author"])
```

Comment:
technically i don't think we even need include author, as we dont show author anywhere....maybe it wouldn;t be bad to show author on recipe detail view so the parse call there would want author include ?